web/html/src/manager/legacy/DateTimePicker.tsx

```diff
@@ -54,15 +54,15 @@ function mountDateTimePickerTo(mountingPoint: HTMLElement | null) {
 
     const onChange = (value: moment.Moment) => {
       // Create a copy of the value in the original UTC offset
-      const adjustedValue = localizedMoment(value).utcOffset(utcOffset);
```

Comment:
If I'm not mistaken, this should make the `utcOffset` variable obsolete.

Reply:
Yeah you're right. I will remove it